packages/perseus-editor/src/widgets/phet-sim-editor.tsx

```diff
@@ -1,45 +1,55 @@
 /* eslint-disable @khanacademy/ts-no-error-suppressions */
 /* eslint-disable react/sort-comp */
-import {Changeable, EditorJsonify} from "@khanacademy/perseus";
+import {LabeledTextField} from "@khanacademy/wonder-blocks-form";
 import * as React from "react";
 
-import BlurInput from "../components/blur-input";
+import type {PerseusPhetSimWidgetOptions} from "@khanacademy/perseus/src/perseus-types";
 
-type PhetSimEditorProps = any;
+type Options = {
+    url: PerseusPhetSimWidgetOptions["url"];
+    description: PerseusPhetSimWidgetOptions["description"];
+};
 
-class PhetSimEditor extends React.Component<PhetSimEditorProps> {
-    static propTypes = {
-        ...Changeable.propTypes,
-    };
-
-    static widgetName = "phet-sim" as const;
+type Props = Options & {
+    onChange: (arg1: {
+        url?: Props["url"];
+        description?: Props["description"];
+    }) => void;
+};
 
-    static defaultProps: PhetSimEditorProps = {
+class PhetSimEditor extends React.Component<Props> {
+    static defaultProps: Options = {
         url: "",
         description: "",
     };
 
-    change: (arg1: any, arg2: any) => any = (...args) => {
-        return Changeable.change.apply(this, args);
-    };
+    static widgetName = "phet-sim" as const;
 
     render(): React.ReactNode {
         return (
             <div>
-                <label>
-                    URL:
-                    <BlurInput
-                        value={this.props.url}
-                        // @ts-expect-error - TS2554 - Expected 2 arguments, but got 1.
-                        onChange={this.change("url")}
-                    />
-                </label>
+                <LabeledTextField
+                    label={"URL"}
+                    value={this.props.url}
+                    onChange={(url: string) => this.props.onChange({url})}
+                />
+                <br />
+                <LabeledTextField
+                    label={"Description"}
+                    value={this.props.description}
+                    onChange={(description: string) =>
+                        this.props.onChange({description})
+                    }
+                />
             </div>
         );
     }
 
-    serialize: () => any = () => {
-        return EditorJsonify.serialize.call(this);
+    serialize: () => Options = () => {
```

Comment:
I realize you're likely following another editor's pattern, but I don't think this needs to be an arrow function. That's only needed if the class method is passed around and we want to ensure `this` stays bound.

```suggestion
    serialize(): PerseusPhetSimWidgetOptions {
```

Reply:
Oh, noted! Thanks. I'd been wondering what the difference was, and was just pattern-matching in the meantime. That makes sense! Thank you!